spec/attribute.dd

```diff
@@ -484,11 +484,9 @@ runtime error.)
 
 ---
 @disable void foo() { }
-
-void main()
-{
-    foo();   // error, foo is disabled
-}
+---
+---
```

Comment:
Same here. We shouldn't make the HTML documentation (which is likely going to be used 1000x more often than the PDF) uglier because of Latex bugs...